tests/kubernetes/services/mysql.yaml

```diff
@@ -26,3 +26,12 @@ spec:
               value: testing123
             - name: MYSQL_DATABASE
               value: testdb
+        - name: mysql-latest
```

Comment:
Run this as two separate pods (which will require two deployments).  I don't even understand how this works because pods share a network namespace so only one of the mysql instances should work since they both bind to port 3306.

Reply:
I'll revert this change. This test will be parametrized for different versions of mysql when the monitor tests are restructured in a future PR.